src/core/pages/structuredMarkdown.tsx

```
@@ -90,21 +110,51 @@ export function splitIntoSections(compiled: JSX.Element[]): React.ReactNode[][]
+ * </section>
+ * ```
+ *
+ * @param {JSX.Element[]} compiled - An array of JSX elements from markdown-to-jsx compiler
+ * @param {string} [idPrefix='hdr'] - Prefix for generated heading IDs
+ * @param {string} [classPrefix='wrap'] - Prefix for wrapper div classes
+ * @returns {React.ReactNode[]} Array of section elements with structured content
  */
-export function structureMarkdownContent(compiled: JSX.Element[]) {
+export function structureMarkdownContent(
+  compiled: JSX.Element[],
+  idPrefix: string = 'hdr',
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)